pkg/crc/api/client/client.go

```diff
@@ -215,14 +215,20 @@ func (c *Client) SetPullSecret(data string) error {
 	return nil
 }
 
-func (c *Client) sendGetRequest(url string) ([]byte, error) {
-	res, err := c.client.Get(fmt.Sprintf("%s%s", c.base, url))
+func (c *Client) sendGetRequest(url string, data io.Reader) ([]byte, error) {
+	req, err := http.NewRequest("GET", fmt.Sprintf("%s%s", c.base, url), data)
+	if err != nil {
+		return nil, err
+	}
+	req.Header.Set("Content-Type", "application/json")
+
+	res, err := c.client.Do(req)
 	if err != nil {
```

Comment:
Isn't this now the same method as `sendPostRequest`?

Reply:
@cfergeau yes, Just the method is different so we can create an helper with same functionality and pass the which method is required `GET/POST ..etc`

Reply:
Yes, I am creating a helper method now.